src/screens/SendConfirmScreen.js

```diff
@@ -95,16 +101,11 @@ class SendConfirmScreen extends React.Component {
 
     // show loading modal
     this.setState({
-      modal:
-        // eslint-disable-next-line react/jsx-indent
-        (<SendTransactionFeedbackModal
-          text={t`Your transfer is being processed`}
-          sendTransaction={sendTransaction}
-          promise={promise}
-          successText={<TextFmt>{t`Your transfer of **${this.amountAndToken}** has been confirmed`}</TextFmt>}
-          onDismissSuccess={this.exitScreen}
-          onDismissError={() => this.setState({ modal: null })}
-        />),
+      modal: {
```

Comment:
We still have this method setting the state in the old format. Should we update it?

```js
  onError = (message) => {
    this.setState({
      modal:
        // eslint-disable-next-line react/jsx-indent
        <FeedbackModal
          icon={<Image source={errorIcon} style={{ height: 105, width: 105 }} resizeMode='contain' />}
          text={message}
          onDismiss={() => this.setState({ modal: null })}
        />,
    });
  }
```

Reply:
`onError` is not being used on `SendConfirm` screen, errors are being handled on the `SendTransactionFeedbackModal` (by listening to `promise` rejections)

But it was being used on `CreateTokenConfirm`, I refactored it on 30bd4d94d73ce47ce41d4a6e0cf8e58b8a9754bf

What do you think?